src/test/librbd/CMakeLists.txt

```diff
@@ -123,6 +123,7 @@ target_link_libraries(ceph_test_librbd_api
 set_target_properties(ceph_test_librbd_api PROPERTIES COMPILE_FLAGS
   ${UNITTEST_CXX_FLAGS})
 
+ifdef(LINUX)
```

Comment:
probably what you meant is
```cmake
if(DEFINED LINUX)
```
but i'd suggest use
```cmake
if(LINUX)
```
for consistency.

but can we just disable the krbd support in `fsx.cc`? is there any other linux specific bits in it?